hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/FactoryIdHelper.java

```diff
@@ -136,9 +136,6 @@
 
     // =========================== portables =============================================
 
-    public static final String SPI_PORTABLE_FACTORY = "hazelcast.serialization.portable.spi";
-    public static final int SPI_PORTABLE_FACTORY_ID = -1;
```

Comment:
since this is removed, should we change the next id `PARTITION_PORTABLE_FACTORY_ID ` to be -1 and so on.